### ISSUE TITLE
Reference Systems subclass of Directive vs Descriptive Information Entity?

### ISSUE THREAD

**Issue by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#147]
Reference Systems are currently a subclass of Descriptive Information Content Entity. I am speculating that they are really a subclass of Directive Information Content Entity under the interpretation that a reference system provides a recipe for creating Designative Information Content Entities. I'm specifically interested in spatial and temporal reference systems at the moment. For example, a spatial coordinate system tells us how to create a set of coordinates (a kind of name/identifier) of a site. 

There are three subclasses of Reference System, currently. I've thought about the spatial and temporal reference systems but not as much as the third subclass, a priority scale.

**Comment by eliasweatherfield** [comment]
> I am speculating that they are really a subclass of Directive Information Content Entity under the interpretation that a reference system provides a recipe for creating Designative Information Content Entities.

I agree that Directive Information Content Entity seems to be a better choice of genus for reference systems.

At first sight, reference systems also seem to be able to be understood as types of a concrete reference. E. g. "12 am" seems to be an instance of "Twelve-Hour Clock Time System reference". And we have at least one other example where something like a type has been represented as a Directive Information Content Entity, namely Artifact Model: "A Directive Information Content Entity that prescribes a common set of Functions and Qualities to inhere in a set of artifact instances."
